File: widget/src/main/java/ru/kest/trainswidget/model/domain/TrainThread.java
package ru.kest.trainswidget.model.domain;

import lombok.Data;

import java.util.Date;

/**
 * Created by KKharitonov on 07.01.2016.
 */
@Data
public class TrainThread {

    private Date arrival;
    private Date departure;
    private String title;

    private String from;
    private String to;

}

File: widget/src/main/java/ru/kest/trainswidget/model/domain/TrainThread.kt
package ru.kest.trainswidget.model.domain

import com.fasterxml.jackson.annotation.JsonCreator
import com.fasterxml.jackson.annotation.JsonProperty
import java.util.*

/**
 * Train Model
 *
 * Created by KKharitonov on 07.01.2016.
 */
data class TrainThread
    @JsonCreator constructor(@JsonProperty("arrival") var arrival: Date,
                             @JsonProperty("departure") var departure: Date,
                             @JsonProperty("title") var title: String,

                             @JsonProperty("from") var from: String? = null,
                             @JsonProperty("to") var to: String? = null
)
